examples/jsm/renderers/WebGLDeferredRenderer.js

```diff
@@ -641,7 +641,7 @@ var WebGLDeferredRenderer = function ( parameters ) {
 
 		}
 
-		updateDeferredColorUniforms( renderer, scene, camera, geometry, material, group );
+		updateDeferredColorUniforms( renderer, scene, camera, geometry, material );
```

Comment:
You have to do this change in the `js` directory and then run the modularize script.

Reply:
done